samples/matter/light_bulb/sysbuild/mcuboot/prj.conf

```diff
@@ -37,3 +35,8 @@ CONFIG_MULTITHREADING=n
 CONFIG_TICKLESS_KERNEL=n
 CONFIG_TIMEOUT_64BIT=n
 CONFIG_NRF_ENABLE_ICACHE=n
+CONFIG_SIZE_OPTIMIZATIONS=y
+
+# Activate LTO
+CONFIG_LTO=y
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
```

Comment:
Is it safe to enable LTO for bootloader? Is this default configuration for L15 as well?

Reply:
Talked with Pluto team and they said we can use LTO now. it is used in nrf_desktop app as well: https://github.com/nrfconnect/sdk-nrf/blob/main/applications/nrf_desktop/configuration/nrf54l15dk_nrf54l15_cpuapp/images/mcuboot/prj.conf#L39